src/phoenix/config.py

```diff
@@ -10,6 +10,7 @@
 ENV_PHOENIX_PORT = "PHOENIX_PORT"
 ENV_PHOENIX_GRPC_PORT = "PHOENIX_GRPC_PORT"
 ENV_PHOENIX_HOST = "PHOENIX_HOST"
+ENV_PHOENIX_ROOT_PATH = "PHOENIX_ROOT_PATH"
```

Comment:
I think the changing env var name `PHOENIX_HOST_ROOT_PATH` is a bit better? We manage all of our configuration via environment variables and `PHOENIX_ROOT_PATH` might sound a little too generic

Reply:
Agreed, should we match the CLI parameter then to also be `--host_root_path` or should we stick with the `--root-path` parameter as it is known from uvicorn, for example?

Reply:
Went with the matching CLI parameter for now.